tests/subsys/event_manager/src/modules/test_oom.c

```diff
@@ -33,6 +33,24 @@ void sys_reboot(int type)
 	}
 }
 
+void *event_manager_alloc(size_t size)
```

Comment:
Maybe we could move them to separate file? They affect whole test, not just this module. Currently they may be quite difficult to find.